### ISSUE TITLE
Add some message when passing stream-URL

### ISSUE THREAD

**Issue by Pohli** [zwieberl/harbour-fishtheke#17]
If I choose to open a video with LLs VideoPlayer on my Jolla Phone Fishtheke returns to video details page and it can take up to a few seconds till VideoPlayer opens noticeable. This irritated me when I did that for the first time. So is it possible to show a message like "Please wait a moment!" while VideoPlayer is starting?
Maybe on other phones the gap is shorter to not existent, I can't say as I only have a Jolla Phone.

Jolla Phone with SFOS 3.0.1.11 and Fishtheke 0.8

**Comment by zwieberl** [comment]
Yes, I see. Other app-integrations (Browser, Gallery, Vodman) would probably have the same problem as well (although I think dbus-integrations like with Vodman are way faster since the app has to be already running). 

The easiest solution would be to send a temporary notification saying "Sending to XY. Please wait" (which would appear in the upper left corner for, say, 3 seconds or so).
Same may be a good idea for "copying to clipboard" and the like. 

Would that be a ok?

**Comment by Pohli** [comment]
Yes, sounds good.
